ansible/backup_rabbitmq.yml

```diff
@@ -0,0 +1,31 @@
+---
+- hosts: rmq-cluster
+  vars:
+    backup_dir: "/path/to/backup" # need to pass this as an extra var
+  tasks:
+    - name: Enable RabbitMQ management plugin
+      command: rabbitmq-plugins enable rabbitmq_management
+      become: true
+
+    - name: Ensure backup directory exists
+      file:
+        path: "{{ backup_dir }}"
+        state: directory
+
+    - name: Export RabbitMQ configurations
+      command: rabbitmqadmin export "{{ backup_dir }}/definitions.json"
+
+    - name: Get RabbitMQ node directory
+      command: rabbitmqctl eval 'rabbit_mnesia:dir().'
+      register: node_directory
+
+    - name: Set RabbitMQ node directory path
+      set_fact:
+        rabbitmq_node_dir: "{{ node_directory.stdout | regex_replace('\"', '') }}"
+
+    - name: List contents of RabbitMQ node directory
```

Comment:
**issue:** it might be necessary to [stop the node](https://www.rabbitmq.com/docs/backup#messages-backup) before backing up the data directory.

**suggestion:** stop the node before even listing the data directory / archiving files. Also ensure the node is restarted after the backup is pulled.